src/manifest-creation.ts

```diff
@@ -58,10 +58,8 @@ export class ManifestCreation {
 
   public async createAppFromCode (code: any) {
     const github = GitHubAPI()
-    const response = await github.request({
-      headers: { accept: 'application/vnd.github.fury-preview+json' },
-      method: 'POST',
-      url: `/app-manifests/${code}/conversions`
+    const response = await github.request(`POST /app-manifests/${code}/conversions`, {
+      headers: { accept: 'application/vnd.github.fury-preview+json' }
```

Comment:
this should better be `POST /app-manifests/:code/conversions` and then pass code as option below `headers`